packages/ui/src/FernsProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import {ToastProvider} from "react-native-toast-notifications";
+
+import {Toast} from "./Toast";
+
+export function FernsProvider({children}: {children: React.ReactNode}): React.ReactElement {
```

Comment:
Will every app that uses the ferns-ui library require this provider?

Reply:
If they want toasts (and others in the future), yeah. I'll add to the docs.